.github/workflows/ci.yml

```diff
@@ -0,0 +1,49 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: 16.x
+    - uses: jetli/wasm-pack-action@v0.3.0
+    - name: npm install and build
+      run: |
+        npm install
+        npm run build:v2-compat
```

Comment:
Use `npm run build:v2-compat -- firefox` for this, otherwise the generated compat layer will be by default be only for firefox.

Reply:
```suggestion
        npm run build:v2-compat -- firefox
```

Reply:
Same thing for the other v2-compat's